### ISSUE TITLE
DualSense controller on Resident Evil 2/3/4 bugs 

### ISSUE THREAD

**Issue by Asais10** [FIX94/Nintendont#1155]
I am using the original DualSense, not DualSense Edge. I am using the controller_ps5.ini included in this repository. 

While playing Resident Evil 4, both R2 and L2 buttons bring out the knife. The intended behavior is that R2 brings out the equipped gun instead, while L2 brings out the knife. 

This happens if I'm using the Knife Type 1 (press L2 to perform a quick slash, when L2 is not pressed the knife is not equipped). 

If I use Knife Type 2 (press L2 to select the knife and then R2 to slash, press L2 again to holster knife) if I first press L2 and then R2 then the gun aims, L2 by itself does nothing yet R2 still brings out the knife. 

The bug in RE3 is yet again about the R2 button . If I select control type C, which should make R2 walk the player forward, it still causes the player to aim like the other control schemes. 

I also tested Resident Evil 2, and there R2 causes the player to aim, yet L2 to change target does not do anything, even when autoaim is enabled. 

This may not even be a bug within Nintendont itself, please recheck ps5_controller.ini first. I kind of tried to fix it myself with HIDTest but I just kind of couldn't get it to work, I did try to switch the R2 code from "9.08" to "8.08" and that causes the player to walk forwards in RE3 with the C control scheme as intended, but always, not just when R2 is pressed down.

**Comment by Asais10** [comment]
Another thing to note is that if even if I slightly press L2 before pressing R2 and then stop pressing L2 while still holding R2 in Resident Evil 4 with the Knife Type 2 it will continue aiming with the gun

**Comment by Asais10** [comment]
Yup it's definitely an issue with the analog triggers of DualSense and the configuration .ini rather than Nintendont itself. In Resident Evil 4 I swapped the Z button (R1) and Right Shoulder (R2) and then R2 still brings out the knife rather than the map which Z button does normally but the R1 now brings out the gun as intended

**Comment by Asais10** [comment]
I have found a way to fix it: set DigitalLR to 1 instead of 2 in the .ini file. Close this issue when the controller_ps5.ini gets that edited.

**Comment by daithi-coyle** [comment]
Will this result in being unable to use a "half press" on the triggers for games like Sunshine?

**Comment by daithi-coyle** [comment]
After some testing:

Set:
`DigitalLR=2`

Change:
```
L=5,ff
R=6,ff
```

Gets full functionality from the shoulder buttons without any L2/R2 conflict

**Comment by Asais10** [comment]
Does anybody know the rumble values for DualSense?

**Comment by Asais10** [comment]
I will close this when controller_ps5.ini gets edited with the fixes by daithi_coyle

**Comment by Asais10** [comment]
Does anybody know the rumble values for DualSense?

**Comment by daithi-coyle** [comment]
> I will close this when controller_ps5.ini gets edited with the fixes by daithi_coyle

It says I need to fork the repository to edit.
I can paste my full body below:

`[DualSense]
VID=054C
PID=0CE6
Polltype=1
DPAD=1
DigitalLR=2
Power=9,10
A=8,40
B=8,20
X=8,80
Y=8,10
Z=9,02
L=5,ff
R=6,ff
S=9,20
Left=8,06
Down=8,04
Right=8,02
Up=8,00
RightUp=8,01
DownRight=8,03
DownLeft=8,05
UpLeft=8,07
StickX=1
StickY=2
CStickX=3
CStickY=4
LAnalog=5
RAnalog=6`


I could never figure out how to get the rumble working. I tried a bunch of things but got nowhere with it.

`Rumble=1
RumbleType=0
RumbleDataLen=05
RumbleDataOn=0x02,0xFF,0xF7,0xFF,0xFF
RumbleDataOff=0x02,0xFF,0xF7,0x00,0x00
RumbleTransferLen=05
RumbleTransfers=1`

**Comment by DaddyVampire** [comment]
hi I use the dualsense edge and when I play a game like nfs most wanted, the button to accelerate is not A or well in my case A would be X. I thought
